config/resolvers/git-resolver-config.yaml

```diff
@@ -22,8 +22,6 @@ metadata:
     app.kubernetes.io/instance: default
     app.kubernetes.io/part-of: tekton-pipelines
 data:
-  # The maximum amount of time a single anonymous cloning resolution may take.
```

Comment:
Do we need to deprecate this prior to removing it ? 🙃 resolvers being beta, we *may* have to follow the 9month rule 😬

Reply:
So we're not actually _removing_ the functionality of `fetch-timeout` - anyone who already has it set in their `git-resolver-config` will still have that timeout apply. What we're doing is (a) setting the default to 0, so that the global timeout applies when `fetch-timeout` isn't explicitly set, and (b) removing it from `git-resolver-config` for new installs, to prevent confusion, since users should be using the global timeout, not the git-specific one.

We could leave it in `git-resolver-config.yaml`, set to 0 by default and adding a `DEPRECATED: use global timeout instead blah blah` comment, if need be, but given that the point of deprecation vs removal is to nudge users to use the replacement while not breaking existing users, I think removing `fetch-timeout` from new installs is perfectly fine.

Reply:
Ah thanks for the explanation, makes sense 👍🏼